docs/predefined-algebras.rst

```diff
@@ -27,9 +27,9 @@ The easiest way to get started with ``clifford`` is to use one of several predef
 * .. module:: clifford.dpga
 
   ``dpga``: Double PGA also referred to as the Mother Algebra, ``Cl(4, 4)``.
-* .. module:: clifford.dcga
+* .. module:: clifford.dg3c
 
-  ``dcga``: Double Conformal Geometric Algebra (DCGA), ``Cl(8, 2)``.
+  ``dg3c``: Double Conformal Geometric Algebra, ``Cl(8, 2)``.
```

Comment:
```suggestion
  ``dg3c``: Double Conformal space for G3, ``Cl(8, 2)``.
```
would match the way the other modules express this. Any wording including "3" is fine by me.

This is perhaps an argument for `g3dc` instead of `dg3c`. It also has the nice property that `import clifford.g<tab>` will reveal it.

Reply:
The problem here is that multiple papers refer to it as Double Conformal Geometric Algebra or even just by the acronym DCGA

Reply:
Also I don't think Double Conformal space for G3 directly captures the concept that this really is two CGAs glued together

Reply:
How about?
Double Conformal Geometric Algebra, effectively two g3c algebras glued together

Reply:
Last one sounds good enough, thanks.